framework/web/IdentityInterface.php

```diff
@@ -102,4 +113,20 @@ public function getAuthKey();
      * @see getAuthKey()
      */
     public function validateAuthKey($authKey);
+
+    /**
+     * Update the "auth key" token for the given user identity.
+     *
+     * @param IdentityInterface $identity
+     * @param string $token
+     * @return void
+     */
+    public function updateAuthKey(IdentityInterface $identity, $token);
```

Comment:
We cannot introduce new methods in this interface, it will be major BC break.

Reply:
I know in current design i didnt find a way to not break BC